### PR TITLE
Reset

### PR DESCRIPTION
**Pull request by bitNathan**
Added reset endpoint for the API

### CODE REVIEW

File: backend/src/routes/reset.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      host: process.env.AWS_RDS_ENDPOINT,
+      user: process.env.AWS_RDS_USERNAME, 
+      password: process.env.AWS_RDS_PASSWORD, 
+      database: process.env.AWS_RDS_DATABASE_NAME,
+      port: parseInt(process.env.AWS_RDS_PORT as string, 10)
+    });
+    // console.log('Database connection established successfully');
+  } catch (error) {
+    // Log the environment variables to ensure they are being read correctly
+    console.log('Database configuration:', {
```
Comment (MadSons) [inline backend/src/routes/reset.ts]: Not sure if we should log some secure passwords to the console.

**Review by MadSons — DISMISSED** [review]
I think it looks good, just the one change with logging passwords and you can merge.

File: backend/src/routes/reset.ts (hunk 1 above)
Comment (MadSons) [reply backend/src/routes/reset.ts]: @bitNathan

Comment (bitNathan) [reply backend/src/routes/reset.ts]: updated, was using it for testing since the SQL database was being difficult last week

**Pull request merged**